### DISCUSSION TITLE
Resume tenosboard logging when resum training

### DISCUSSION THREAD

**Discussion by talhaanwarch** [Lightning-AI/pytorch-lightning#8451] [General]
When resume training in trainer, how to resume tensorboard logging. i think tensorboard will start logging from 0, instead of logging from where it ends

**Comment by Keiku** [comment]
Check the log by collecting all events.out.tfevents of the TensorBoard log in the same folder.

I was able to aggregate the logs as shown in the link below.

https://github.com/PyTorchLightning/pytorch-lightning/issues/5325#issuecomment-881827435

**Reply by talhaanwarch** [reply]
thanks, i think it works

**Reply by Keiku** [reply]
@talhaanwarch If you have solved it, please check Answered so that other people can understand it.
